src/status_im/ui/screens/discover/styles.cljs

```diff
@@ -165,9 +165,11 @@
    :background-color styles/color-white})
 
 (def all-dapps-flat-list-item
-  {:margin 10
-   :width  100
-   :height 100})
+  {:margin          10
+   :width           100
```

Comment:
Will those adapt smaller/ bigger screens?

Reply:
Good question, I think it should be fine but might have to set 2 or 4 rows for other resolutions. Will check in further PRs